CMakeLists.txt

```diff
@@ -115,6 +115,15 @@ foreach(D IN LISTS AMReX_SPACEDIM)
         $<BUILD_INTERFACE:${pyAMReX_SOURCE_DIR}/src>
     )
 
+    if(${AMReX_MPI})
+      execute_process(COMMAND
+        ${Python_EXECUTABLE} -c "import mpi4py; print( mpi4py.get_include() )"
```

Comment:
As a note, this does not work when cross-compiling. I will work on an alternative that does not needs mpi4py to be present and executable at compile time.

- https://github.com/openPMD/openPMD-api/blob/c2e0e83f0380fad0a7996cb6a496d0f66d2d5b6b/src/binding/python/Series.cpp#L38-L52
- https://github.com/openPMD/openPMD-api/blob/c2e0e83f0380fad0a7996cb6a496d0f66d2d5b6b/src/binding/python/Series.cpp#L226-L308

Reply:
Thanks, Axel :+1: ! I included the dependence on mpi4py as my project involves a downstream task that depends on mpi4py. Actually getting just the C++ version of MPI_Comm from https://github.com/AMReX-Codes/amrex/blob/development/Src/Base/AMReX_MPMD.cpp#L97 and directly passing it back to amrex::Initialize worked in my initial attempts.

Reply:
Hi @siddanib,

Yes the approach I linked will work with `mpi4py` :)
It just changes the build process to build without having to provide a run-able `mpi4y` at configuration and compilation time, which is needed when cross-compiling for different architectures.